tests/karate/src/test/java/fe/utils/android.feature

```diff
@@ -34,10 +41,16 @@ Feature: android page object
       # If this breaks, use this code to log the page hierarchy :
       # karate.log(driver.getHttp().path("source").get().value)
 
-      And click('//*[@content-desc="More options"]')
-      And click('#com.github.dedis.popstellar:id/title')
+    And click('//*[@content-desc="More options"]')
+    And click('#com.github.dedis.popstellar:id/title')
 
       # Input the mock backend url and connect to it
-      And input('#com.github.dedis.popstellar:id/entry_box_server_url', backendURL)
-      And click('#com.github.dedis.popstellar:id/button_apply')
-      And match backend.waitForConnection(5000) == true
+    And input('#com.github.dedis.popstellar:id/entry_box_server_url', backendURL)
+    And click('#com.github.dedis.popstellar:id/button_apply')
+    And match backend.waitForConnection(5000) == true
+
+    And click(tab_wallet_selector)
+    And click(tab_wallet_new_wallet_selector)
+    And click(tab_wallet_confirm_selector)
+    And dialog(true)
```

Comment:
Having a lot of Ands like this is not very informative IMO, better use some other conjunctions like Then and When, to make it more readable. This has no impact on the functionality I just think that it is more clear for newcomers who try to understand the code. If we only want to perform scripting without trying to satisfy the reader I think going with * instead of And is a better option.

Reply:
I agree I've updated it and the existing code